infra/src/main/kotlin/fi/fta/geoviite/infra/geometry/GeometryDao.kt

```diff
@@ -39,17 +39,15 @@ enum class VerticalIntersectionType {
 @Component
 class GeometryDao @Autowired constructor(
     jdbcTemplateParam: NamedParameterJdbcTemplate?,
-    private val kkJtoETRSTriangulationDao: KKJtoETRSTriangulationDao,
 ) : DaoBase(jdbcTemplateParam) {
 
     @Transactional
     fun insertPlan(
         plan: GeometryPlan,
         file: InfraModelFile,
+        boundingBoxInLayoutCoordinates: List<Point>?,
```

Comment:
Ideana se, että BoundingBox (sikäli kun sellainen löytyy) muunnetaan TM35FIN:iin jo ennen inserttiä. Tällöin päästään eroon riippuvuudesta `kkjToETRSTriangulationDao`:on